src/tests/option_round/options_sold_tests.cairo

```diff
@@ -105,65 +111,54 @@ fn test_total_options_after_auction_2() {
 #[available_gas(10000000)]
 fn test_total_options_after_auction_3() {
     let (mut vault_facade, _) = setup_facade();
-    // LP deposits (into round 1)
-    let deposit_amount_wei: u256 = 10000 * decimals();
-    vault_facade.deposit(deposit_amount_wei, liquidity_provider_1());
-    // Start auction
-    vault_facade.start_auction();
-    let mut round_facade: OptionRoundFacade = vault_facade.get_current_round();
-    let params: OptionRoundParams = round_facade.get_params();
-    // OB 1 and 2 bid for > the total options available at the reserve price
```

Comment:
The comment is wrong in this test, but we should use a custom accelerator to match its behavior. This should use accelerate_to_auctioning_running_custom() not the default

Reply:
aren't we gonna change the default accelerator so that it fetches the default values and then internally call the custom function?